### PR TITLE
MINOR: Update offset.storage.topic description

### PR DESCRIPTION
**Pull request by fvaleri**
This is a minor change just to clarify that this topic only stores offsets of source connector tasks. Instead, offsets for sink connector tasks are stored in __consumer_offsets, as for consumer groups.

### CODE REVIEW

**Comment by fvaleri** [comment]
@mimaison

File: connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConfig.java [hunk 1]
```diff
@@ -108,7 +108,7 @@ public class WorkerConfig extends AbstractConfig {
 
     public static final String OFFSET_COMMIT_INTERVAL_MS_CONFIG = "offset.flush.interval.ms";
     private static final String OFFSET_COMMIT_INTERVAL_MS_DOC
-            = "Interval at which to try committing offsets for tasks.";
+            = "Interval at which to try committing offsets for source tasks.";
```
Comment (C0urante) [inline connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConfig.java]: The offset commit interval also affects sink tasks; see here:
https://github.com/apache/kafka/blob/352c71ffb5d825c4a88454c12b9fa66c1750add3/connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSinkTask.java#L212
We can probably revert this change.

File: connect/runtime/src/main/java/org/apache/kafka/connect/runtime/standalone/StandaloneConfig.java [hunk 2]
```diff
@@ -28,7 +28,7 @@ public class StandaloneConfig extends WorkerConfig {
      * <code>offset.storage.file.filename</code>
      */
     public static final String OFFSET_STORAGE_FILE_FILENAME_CONFIG = "offset.storage.file.filename";
-    private static final String OFFSET_STORAGE_FILE_FILENAME_DOC = "File to store offset data in";
+    private static final String OFFSET_STORAGE_FILE_FILENAME_DOC = "File to store source task offsets";
```
Comment (C0urante) [inline connect/runtime/src/main/java/org/apache/kafka/connect/runtime/standalone/StandaloneConfig.java]: To remain consistent with the description for the `offset.storage.topic`, we should probably use the term "connector" instead of "task":
```suggestion
    private static final String OFFSET_STORAGE_FILE_FILENAME_DOC = "File to store source connector offsets";
```

File: docs/connect.html [hunk 3]
```diff
@@ -53,7 +53,7 @@ <h4><a id="connect_running" href="#connect_running">Running Kafka Connect</a></h
 
     <p>The important configuration options specific to standalone mode are:</p>
     <ul>
-        <li><code>offset.storage.file.filename</code> - File to store offset data in</li>
+        <li><code>offset.storage.file.filename</code> - File to store source task offsets</li>
```
Comment (C0urante) [inline docs/connect.html]: Same suggestion RE "connector" vs. "task":
```suggestion
        <li><code>offset.storage.file.filename</code> - File to store source connector offsets</li>
```

**Comment by C0urante** [comment]
Thanks @fvaleri, this is a useful change. I've left a few small comments but it should be good to go after they're addressed.

File: docs/connect.html (hunk 3 above)
Comment (fvaleri) [reply docs/connect.html]: Thanks.

File: connect/runtime/src/main/java/org/apache/kafka/connect/runtime/standalone/StandaloneConfig.java (hunk 2 above)
Comment (fvaleri) [reply connect/runtime/src/main/java/org/apache/kafka/connect/runtime/standalone/StandaloneConfig.java]: Yes, I like consistency. Thanks.

File: connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConfig.java (hunk 1 above)
Comment (fvaleri) [reply connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConfig.java]: Right, thanks.

**Comment by fvaleri** [comment]
Thanks @C0urante for the quick feedback. It should be good now.

**Review by C0urante — APPROVED** [review]
LGTM, thanks @fvaleri! Will merge if the build completes without any related failures.
